TesseractOCR/G8Tesseract.mm

```diff
@@ -310,18 +340,124 @@ - (void)setVariablesFromDictionary:(NSDictionary *)dictionary
 
 - (void)loadVariables
 {
-    [self.variables enumerateKeysAndObjectsUsingBlock:^(NSString *key, NSString *value, BOOL *stop) {
-        _tesseract->SetVariable(key.UTF8String, value.UTF8String);
-    }];
+    if (self.isEngineConfigured) {
+        [self.variables enumerateKeysAndObjectsUsingBlock:^(NSString *key, NSString *value, BOOL *stop) {
+            _tesseract->SetVariable(key.UTF8String, value.UTF8String);
+        }];
+    }
+}
+
+#pragma mark - Internal getters and setters
+
+- (tesseract::TessBaseAPI *)tesseract {
+    
+    if (!_tesseract) {
+        _tesseract = new tesseract::TessBaseAPI();
+    }
+    return _tesseract;
+}
+
+- (void)setEngineImage:(UIImage *)image {
+    
+    if (image.size.width <= 0 || image.size.height <= 0) {
+        NSLog(@"ERROR: Image has not size!");
```

Comment:
I know you just moved this code, but can you change this log message to be "Image has invalid size!"?